ios/iosApp/sample/HomeViewController.swift

```diff
@@ -0,0 +1,8 @@
+import TrikotFrameworkName
+import UIKit
+
+class HomeViewController: BaseViewModelViewController<HomeView, HomeViewModelController, HomeViewModel> {
+    init() {
+        super.init(viewModelController: Core.shared.viewModelControllerFactory.createHome())
+    }
```

Comment:
For the same reason stated in the Android portion, it would make more sense I think to pass the VMC as a constructor parameter directly in a `ViewControllerFactory`

Reply:
100% agrees for iOS 😉